test/Microsoft.NET.Sdk.Functions.Generator.Tests/DisableAttributeTests.cs

```diff
@@ -53,8 +53,8 @@ public class FunctionsClass4
         [InlineData(typeof(FunctionsClass3), "Run", false)]
         public void MethodsWithDisabledParametersShouldBeDisabled(Type type, string methodName, object expectedIsDisabled)
         {
-            var method = type.GetMethod(methodName);
-            var funcJson = method.ToFunctionJson(string.Empty);
+            var methodDef = TestUtility.GetMethodDefinition(type, methodName);
```

Comment:
nit: I know you were just updating the code, but since you're changing the variable name, let's change from `var` to the type name here.